### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**
Updates based on https://github.com/mattermost/mattermost-webapp/blob/release-7.9/package.json.

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -3976,77 +4047,6 @@ If the Work includes a "NOTICE" text file as part of its distribution, then any
 
 END OF TERMS AND CONDITIONS
 
-
----
-
-## webpack
```
Comment (m1lt0n) [inline NOTICE.txt:3982]: (question) why is webpack removed? is it not a requirement to include their licence anymore?

**Review by m1lt0n — APPROVED** [review]
Only 1 comment, but I've approved.

File: NOTICE.txt (hunk 1 above)
Comment (amyblais) [reply NOTICE.txt:3982]: I didn't see it in the package.json, but I've added it back now.

**Review by hmhealey — APPROVED** [review]

**Pull request merged**